buildsystem/templates/openage.bat.in

```diff
@@ -0,0 +1,10 @@
+@rem Copyright 2017-2017 the openage authors. See copying.md for legal info.
+@echo off
+
+rem @AUTOGEN_WARNING@
+
+set INST_DIR=%CD%
+set PATH=%INST_DIR%\@CMAKE_INSTALL_BINDIR@;%INST_DIR%\@CMAKE_PY_INSTALL_PREFIX@
+set XDG_DATA_HOME=%INST_DIR%\share
+
+python.exe @CMAKE_INSTALL_BINDIR@\openage
```

Comment:
doesn't it need the execute-module switch `-m` and the install bindir added to `PYTHONPATH`?

Reply:
If I try running with `python -m openage`, I get:
`NameError: name 'exit' is not defined` from `__main__.py`.
I can address that by adding a `from sys import exit` statement (like the one below) to that file also.

Instead of setting `PYTHONPATH`, having the `python36._pth` (already available in the python embed zip) is sufficient. It allows using the python modules from 'python36.zip', and the same directory. I've written the scripts such that they install the required modules to the latter, so everything works out without the environment variable. [Can't say the same about the other variables I set in the batch file. I'm open to suggestions which can help remove those also.]